android/app/src/main/res/values/strings.xml

```diff
@@ -2263,4 +2263,5 @@
     <string name="type.amenity.dojo">Dojo</string>
     <!-- https://wiki.openstreetmap.org/wiki/Tag:leisure=sports_hall -->
     <string name="type.leisure.sports_hall">Sports hall</string>
+    <string name="last_selected_tab">lastSelectedTab</string>
```

Comment:
Please add a constant into android/app/src/main/res/values/donottranslate.xml, this file is automatically regenerated by scripts.